internal/handlers/common/aggregations/operators/operators.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package operators provides aggregation operators.
+package operators
+
+import (
+	"context"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+)
+
+// newOperatorFunc is a type for a function that creates an operator.
+type newOperatorFunc func(expression *types.Document) (Operator, error)
+
+// Operator is a common interface for aggregation operators.
+type Operator interface {
```

Comment:
Maybe we should leave it as an `Accumulator` interface for now since we don't know much about future operators' behaviour.

Reply:
Let's keep as an Accumulator for now. Updated